src/types/decimal/mod.rs

```diff
@@ -202,83 +198,9 @@ impl Decimal {
         // d1 has the larger exponent (3). We need to scale its coefficient up to d2's 10^2 scale.
         // We do this by multiplying it times 10^exponent_delta, which is 1 in this case.
         // This lets us compare 80 and 80, determining that the decimals are equal.
-        let mut scaled_coefficient: BigUint = d1.coefficient.magnitude().to_biguint().unwrap();
-        scaled_coefficient *= BigUint::from(10u64).pow(exponent_delta as u32);
-        UInt::from(scaled_coefficient).cmp(d2.coefficient.magnitude())
-    }
-
-    /// Extract the integer and fractional parts and the exponents of this as a `(BigInt, (BigInt, i64))`
-    ///
-    /// ```ignore
-    /// # use num_bigint::BigInt;
-    /// # use ion_rs::Decimal;
-    /// let d1: Decimal = Decimal::new(123456789, -2);
-    /// let (int_part, (frac_part, exponent)) = d1.into_parts();
-    /// assert_eq!(int_part, BigInt::from(1234567u64));
-    /// assert_eq!(frac_part, BigInt::from(89));
-    /// assert_eq!(exponent, -2);
-    /// ```
-    pub(crate) fn into_parts(self) -> (BigInt, (BigInt, i64)) {
-        let magnitude: BigInt = self.coefficient.try_into().unwrap();
-        if self.exponent.is_zero() {
-            (magnitude, (BigInt::zero(), 0))
-        } else if self.exponent.is_negative() {
-            let divisor = BigInt::from(10u64).pow((-self.exponent) as u32);
-            let (i, f) = magnitude.div_rem(&divisor);
-            (i, (f, self.exponent))
-        } else {
-            let multiplicand = BigInt::from(10u64).pow(self.exponent as u32);
-            (magnitude * multiplicand, (BigInt::zero(), 0))
-        }
-    }
```

Comment:
🗺️ This and related methods existed solely to enable conversion of decimal components to/from `BigInt`.